### PR TITLE
Delete unused files and images

### LINKED ISSUE #1079: Delete all of the team page files
We have a bunch of unused images and we are not using the team page. The images are of officers from back in 2018-19-20. The team page code is also hardcoded which is just bad practice. [`Core-v4/src/Pages/TheTeam/TheTeam.js`](https://github.com/SCE-Development/Core-v4/blob/dev/src/Pages/TheTeam/TheTeam.js) But yeah this has got to go 😔

Later on, we can implement it using maybe S3 and/or DB, which is more efficient.

### PR DESCRIPTION
**Pull request by neelp03**
Resolves #1079 

- Removes dozens of unused images
- Removes code and tests for the team page

### CODE REVIEW

File: src/Components/Footer/footer.css [hunk 1]
```diff
@@ -1,12 +1,38 @@
+@import url('https://fonts.googleapis.com/css2?family=Mina&family=Montserrat:wght@800&display=swap');
+
 .footer-text p {
```
Comment (evanugarte) [inline src/Components/Footer/footer.css:3]: are these changes related?

File: src/Components/Footer/Footer.js [hunk 2]
```diff
@@ -8,56 +8,132 @@ export default () => {
       <Row className='footer-text'>
         <Col>
           <b>Software & Computer Engineering Society</b>
-          <p>San Jose State University</p>
-          <p>Charles W. Davidson College of Engineering</p>
-          <p>Room: ENGR 294</p>
+          <ul>
```
Comment (evanugarte) [inline src/Components/Footer/Footer.js:11]: same here, idk if this pr was just removing `Pages/TheTeam`

File: src/Components/Footer/footer.css (hunk 1 above)
Comment (neelp03) [reply src/Components/Footer/footer.css]: actually, they are not. I accidentally carried over my changes from the footer update pr here. I fixed it.

**Review by evanugarte — APPROVED** [review]
Legend, I wonder how much this affects the docker image size

**Review by c00lme — APPROVED** [review]
poggerz
